Lesson_2/Lesson_2.js

```diff
@@ -0,0 +1,154 @@
+ // Задача 1
+ var a = 1, b = 1, c, d;
+ c = ++a; alert(c); alert(a); // c=2 префикс значение операнда возвращается увеличенным на единицу. 
+ d = b++; alert(d); alert(b); // d=1 постфикс значение операнда возвращает старое значение b=1, а затем b увеличивается на единицу b=2.
+ c = (2+ ++a); alert(c); alert(a); // c=5; a увеличили до 3 и прибавили 2
+ d = (2+ b++); alert(d); alert(b); // d=4; b в значении 2 сложили с 2, затем увеличили до тройки для следующих вычислений или для вывода
+ alert(a);                    // 3
+ alert(b);                    // 3
+
+ // Задача 2
+ var a = 2;
+ var x = 1 + (a *= 2); // (*=) Умножение, совмещённое с присваиванием, иначе a=a*2
+ console.log(a); // 4
+console.log(x); //  5
+
+// Задача 3 
+ var a = +prompt("Введите любое целое число:");
+ var b = +prompt("Введите любое целое число:");
+ if(isNaN(a && b)) {
+     alert("Введены неверные значения!"); 
+ }
+  else if (a > 0 && b > 0) {
+ x = a - b;
+ alert(x);
+ } 
+ else if (a < 0 && b < 0) {
+ x = a * b;
+ alert (x);
+ }
+ else if (a > 0 && b < 0 || a < 0 && b > 0) {
```

Comment:
Здесь можно просто else, результат будет тот же. Еще небольшой стилистический момент - не стоит переносить else на другую строчку, пусть будет на одной строчке с закрывающей скобочкой
```suggestion
 else {
```